app/common/helpers/validators/uuid-validate.js

```diff
@@ -0,0 +1,18 @@
+import isUuidValid from './uuid';
+
+/**
+ * We need this wrapper because:
+ * 1. Redux-form has strange logic, we need to return undefined if value is valid.
+ * 2. We need to rule error's message.
```

Comment:
we need to rule? hah?

Reply:
may be better 'we must have control over error message'

Reply:
Thx dude. It was my "kievskiy angliuskiy" 😀